go/iot/protocol/Protocol.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package protocol
+
+const (
+	OK                = 200
+	FAIL              = 404
+	UNREGISTERTAG     = 501
```

Comment:
I think this case is still 200, per your doc: https://github.com/census-ecosystem/opencensus-experiments/blob/master/go/iot/protocol/doc/protocol.txt#L56-L58?

Reply:
forget to update the doc. I would update it.